data/meeting_minutes.yml

```diff
@@ -279,6 +279,14 @@ items:
        
        
    specification_committee:
+       -
```

Comment:
These 2 entries are reversed, and appear reversed on the site. These aren't currently sorted outside of by year.

Reply:
Many thx for your help Martin!!!!